torchmetrics/functional/text/bleu.py

```diff
@@ -54,6 +54,19 @@ def _bleu_score_update(
     ref_len: Tensor,
     n_gram: int = 4
 ) -> Tuple[Tensor, Tensor]:
+    """
+    Updates and returns variables required to compute the BLEU score.
+
+    Args:
+        reference_corpus: An iterable of iterables of reference corpus
+        translate_corpus: An iterable of machine translated corpus
+        numerator:
```

Comment:
Need some help here, describing these variables. I tried to understand the context and google a few things, but could not come with concise descriptions.

Reply:
@sameetsinghsidhu mind check :]